newIDE/app/scripts/lib/DokuwikiHelpers.js

```diff
@@ -1,30 +0,0 @@
-// @ts-check
-
-/**
- * Convert the markdown text into markdown supported by DokuWiki, converting
- * the links and image references.
- * @param {string} markdownText
- * @returns {string}
- */
-const convertMarkdownToDokuWikiMarkdown = markdownText => {
-  const markdown = markdownText
-    // Replace images (`![label](image url)`)
-    .replace(/\!\[(.*?)\]\((.*?)\)/g, (match, linkText, linkUrl) => {
-      const url = linkUrl.replace(/^\/+/, '');
-      const title = linkText.replace(/^\[(.*?)\]/, '$1');
-      return `{{ ${url}?nolink |}}`;
-    })
-    // Replace links (`[text](url)`)
-    .replace(/\[(.*?)\]\((.*?)\)/g, (match, linkText, linkUrl) => {
-      const url = linkUrl.replace(/^\/+/, '');
-      const title = linkText.replace(/^\[(.*?)\]/, '$1');
-      return `[[${url}|${title}]]`;
-    })
-    // Add a new line before each list, to make sure DokuWiki renders it correctly.
-    .replace(/((\n[-\*].*)+)/gm, '\n$1')
```

Comment:
@AlexandreSi @D8H Seems we still need this to fix pages like https://wiki.gdevelop.io/gdevelop5/extensions/fire-bullet/

For the pages we manually write, it's "easy" to check the preview generated by Vercel.
For the markdown coming from extensions, we have to fix it manually because it renders well in the editor already, so we can't fix all extensions descriptions (someone will make the mistake one day or the other)

Reply:
GDevelop UI doesn't allow user to see how the md is rendered before its merged (but maybe it should).